xcodeproj/internal/automatic_target_info.bzl

```diff
@@ -99,6 +99,7 @@ _OBJC_LIBRARY_XCODE_TARGETS = {
 }
 _SWIFT_LIBRARY_XCODE_TARGETS = {
     "deps": _XCODE_TARGET_TYPES_COMPILE_AND_NONE,
+    "macros": _XCODE_TARGET_TYPES_COMPILE_AND_NONE,
```

Comment:
Draft, because this will become `plugins` with https://github.com/bazelbuild/rules_swift/commit/b1147292d97fa24b2526cea88034fe1d122e4f36.